packages/contracts/test/Attesting.test.ts

```diff
@@ -156,13 +141,9 @@ describe('Attesting', () => {
         expect(await unirepContract.collectedAttestingFee()).to.be.equal(0)
         const tx = await unirepContract
             .connect(attester)
-            .submitAttestation(
-                attestation,
-                epochKey as BigNumberish,
-                epochKeyProofIndex,
-                senderPfIdx,
-                { value: attestingFee }
-            )
+            .submitAttestation(attestation, epochKey as BigNumberish, {
```

Comment:
we should let `genEpochKey()` returns `bigint` instead of `BigInt` in the future.
(can be in another PR)